src/main/java/Deadline.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * The Deadline class extends the Task class as it is a more specific type of task.
+ */
+public class Deadline extends Task {
+
+    private String deadline;
```

Comment:
Personal preference, but since `deadline` does not get updated, we can add the `final` keyword to it. This helps to signal to other developers that this field should be immutable (unless you are implementing a method to change this). 

Since this isn't in the coding standards, feel free to close this if you don't agree with me